src/styles/sidebar/components/menu-item.scss

```diff
@@ -143,17 +134,26 @@ $menu-item-padding: 10px;
   border-radius: 12px;
 
   &:hover {
-    background-color: var.$grey-3;
-    color: var.$grey-5;
+    background-color: var.$grey-4;
+    color: var.$grey-6;
   }
 
   &-icon {
     width: 12px;
     height: 12px;
   }
+
+  svg {
```

Comment:
Is this not covered by the `&-icon` rule above?

Reply:
Ya, this is still needed, but I think it can be refactored so its simpler and less confusing.